apps/metadata-editor/src/app/dashboard/sidebar/sidebar.component.ts

```diff
@@ -28,10 +31,18 @@ export class SidebarComponent implements OnInit {
   public placeholder$ = this.avatarService.getPlaceholder()
   organisations$: Observable<Organization[]>
 
+  get settingsUrl() {
+    return (
+      this.authService.loginUrl.split('/catalog')[0] +
+      '/admin.console#/organization/users?userOrGroup='
```

Comment:
Would it make sense to create a constant (e.g. `SETTINGS_URL`) for this in the auth service like  `LOGIN_URL`?

I'm also wondering where exactly the url should point at. This looks very specific with the users. It is also not really clear in the ticket. I would suggest a more general path like `/geonetwork/srv/eng/admin.console#/settings`.
Another question that arises to me: Is it always `admin.console#` in the url even if the user is not an admin?

Reply:
The ticket said to either create a constant or forge from the login url, and I took the second option because I figured that it needed less manipulation that login/logout urls. But that's just personal opinion, it can be changed :slightly_smiling_face: 

For the other two questions, I don't know honestly. I tried to look into gn for a while and that's the URL that made more sense, so I used it. 

@LHBruneton-C2C I had more or less the same interrogations during the dev. I did what made most sense to me but I knew it would need some changes & some more infos for that.

Reply:
Ah yes you are right! You don't have to change it :-)

Reply:
Looking at how you need to split from the login URL, and not just complete it, I'd rather go for a specific SETTINGS_URL config key. But that's just an opinion here. This would also solve the second question, as the whole URL would be configurable.

Reply:
Ok good :) Well I think as a user, I would be expecting to land on a page where I'm directly accessing my own parameters. Instead here, the user ends up on a page where they have to select their username to edit their account parameters. 
But I guess I was expecting something that doesn't exist yet :smile:

Reply:
+1 for a SETTINGS_URL token, keep in mind that this url will be completely different on georchestra for instance 🙂

Reply:
Okay, will do :slightly_smiling_face: 
It would be nice to have that decided in the ticket beforehand next time though! :wink:

Reply:
100% agreed, that was an oversight